values.yaml

```diff
@@ -195,9 +195,17 @@ server:
   # The created route will be of type passthrough
   route:
     enabled: false
-    labels: {}
     annotations: {}
-    host: chart-example.local
+    labels: {}
+    # Host name for the Route
+    host: ""
+    tls:
+      # If `true`, TLS is enabled for the Route
+      enabled: true
+      # Insecure edge termination policy of the Route. Can be `None`, `Redirect`, or `Allow`
+      insecureEdgeTerminationPolicy: Redirect
+      # TLS termination of the route. Can be `edge`, `passthrough`, or `reencrypt`
+      termination: edge
```

Comment:
It's not suggested to use edge or reencrypt TLS termination with Vault, the server should be handling all TLS termination (hence the passthrough).

Reply:
Well, I got your point.
But we can allow options for the other kinds of termination. The usecases can be various. For our case, we manage the TLS cert centrally and all tls are terminated on route level.

Reply:
@jasonodonnell Well, I got your point.
But we can allow options for the other kinds of termination. The usecases can be various. For our case, we manage the TLS cert centrally and all tls are terminated on route level.

Reply:
@jasonodonnell by any chance this PR will be accepted?

Reply:
Yes, I think we can accept this, but we should make the default `passthrough`, since that's been the default for some time now.